lib/python/vdm/server/HTTPListener.py

```diff
@@ -499,6 +499,127 @@ def get_volt_jar_dir():
     return os.path.realpath(os.path.join(Global.MODULE_PATH, '../../../..', 'voltdb'))
 
 
+def get_port(ip_with_port):
+    if ip_with_port != "":
+        if ":" not in ip_with_port:
+            return ip_with_port
+        else:
+            arr = ip_with_port.split(":")
+            return arr[1]
+
+
+def check_port_valid(port_option, servers):
+    if port_option == "http-listener":
+        default_port = "8080"
+    if port_option == "admin-listener":
+        default_port = "21211"
+    if port_option == "zookeeper-listener":
+        default_port = "7181"
+    if port_option == "replication-listener":
+        default_port = "5555"
+    if port_option == "client-listener":
+        default_port = "21212"
+    if port_option == "internal-listener":
+        default_port = "3021"
+    if port_option not in request.json:
+        port = default_port
+        server_port = get_port(servers[port_option])
+        if server_port is None:
+            server_port = default_port
+    else:
+        port = get_port(request.json[port_option])
+        server_port = get_port(servers[port_option])
+    if port == server_port:
+        return jsonify(success=False, errors="Duplicate %s for same hostname" %port_option)
```

Comment:
Show the server name and port as well in the error message. So, something like:
Port <port> for the same host is already used by server <servername> for <port_option>